src/components/RadioButtonWithLabel.tsx

```diff
@@ -1,84 +1,70 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import React, {ComponentType} from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
 import styles from '@styles/styles';
 import FormHelpMessage from './FormHelpMessage';
 import * as Pressables from './Pressable';
 import RadioButton from './RadioButton';
 import Text from './Text';
 
-const propTypes = {
+type RadioButtonWithLabelProps = {
     /** Whether the radioButton is checked */
-    isChecked: PropTypes.bool.isRequired,
+    isChecked: boolean;
 
     /** Called when the radioButton or label is pressed */
-    onPress: PropTypes.func.isRequired,
+    onPress: () => void;
 
     /** Container styles */
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    style?: StyleProp<ViewStyle>;
 
     /** Text that appears next to check box */
-    label: PropTypes.string,
+    label: string;
 
     /** Component to display for label */
-    LabelComponent: PropTypes.func,
+    LabelComponent?: ComponentType;
 
     /** Should the input be styled for errors  */
-    hasError: PropTypes.bool,
+    hasError?: boolean;
 
     /** Error text to display */
-    errorText: PropTypes.string,
-};
-
-const defaultProps = {
-    style: [],
-    label: undefined,
-    LabelComponent: undefined,
-    hasError: false,
-    errorText: '',
+    errorText?: string;
 };
 
 const PressableWithFeedback = Pressables.PressableWithFeedback;
 
-function RadioButtonWithLabel(props) {
-    const LabelComponent = props.LabelComponent;
+function RadioButtonWithLabel({LabelComponent, style, label, hasError = false, errorText = '', isChecked, onPress}: RadioButtonWithLabelProps) {
     const defaultStyles = [styles.flexRow, styles.alignItemsCenter];
-    const wrapperStyles = _.isArray(props.style) ? [...defaultStyles, ...props.style] : [...defaultStyles, props.style];
 
-    if (!props.label && !LabelComponent) {
+    if (!label && !LabelComponent) {
         throw new Error('Must provide at least label or LabelComponent prop');
     }
     return (
         <>
-            <View style={wrapperStyles}>
+            <View style={[...defaultStyles, style]}>
```

Comment:
```suggestion
            <View style={[defaultStyles, style]}>
```

I think we don't need to spread here.

Reply:
removed